code/modules/clothing/spacesuits/rig/suits/civilian.dm

```diff
@@ -5,6 +5,7 @@
 	armor = list(melee = 10, bullet = 10, laser = 10, energy = 10, bomb = 10, bio = 10, rad = 10)
 	offline_slowdown = 1
 	online_slowdown = 0
+	acid_resistance = 1
```

Comment:
Revert this change; the reason being that this value doesn't work in isolation. It only affects the acid damage that this rig and its pieces take. And since it has no pieces which cover the body, it will absorb relatively little, we just won't want rigs getting easily melted